src/Microsoft.Win32.Registry/src/Configurations.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp-Windows_NT;
-      netcoreapp-Unix;
+      netcoreapp2.0-Windows_NT;
+      netcoreapp2.0-Unix;
       netstandard;
```

Comment:
I agree with not packaging netcoreapp2.1 but we are going to need to ensure that when building the netcoreapp vertical we don't require restoring any netcoreapp2.0 packages to produce that build. Otherwise we will be hindering our build-from-source efforts.